.env.sample

```diff
@@ -1,8 +1,6 @@
 # This is a sample .env file for use in local development.
```

Comment:
@samuel-shyaka-dus remember to all newly created and used environment variables in this file to ensure that all team members have their project setup correctly. Apart from that you can also mention them in the pre-requistes.

Those variables I was expecting include:
```js
JWT_EXPIRATION
// others here
```